hircluster.c

```diff
@@ -1191,6 +1184,11 @@ dict *parse_cluster_nodes(redisClusterContext *cc, char *str, int str_len,
         }
     }
 
+    if (slot_ranges_found == 0) {
+        __redisClusterSetError(cc, REDIS_ERR_OTHER, "No slot information");
+        goto error;
+    }
```

Comment:
Is this error message is new? Until now we just accepted that we don't have any slots? If so, I think it deserves mention in the commit message.